packages/nextjs/hooks/scaffold-eth/useScaffoldContractWrite.tsx

```diff
@@ -34,9 +36,15 @@ export const useScaffoldContractWrite = (contractName: string, functionName: str
       return;
     }
 
+    // Looking at blockchain to see whats stored at `deployedContractData.address`, if its `0x0` then high possibility that its not a contract
+    const code = await provider.getCode(deployedContractData.address);
+    if (code === "0x") {
```

Comment:
is it '0x' or '0x0' like in the comment?

Reply:
Tysm @rin-st, I think in hex `0x` and `0x0` are same but might be wrong so fixed the comment at [f326e6e](https://github.com/scaffold-eth/se-2/pull/130/commits/f326e6ed4b9f4455de22d52d3f350d43025dbfd9), also while comparing its compared with `0x` as suggested here : https://github.com/ethers-io/ethers.js/discussions/3084#discussioncomment-2954385